src/services/ParticipantProjectSpeciesService.ts

```diff
@@ -0,0 +1,118 @@
+import { components } from 'src/api/types/generated-schema';
+
+import { Response } from './HttpService';
+
+export type ProjectSpecies = {
```

Comment:
Can you rename this to `ParticipantProjectSpecies`?

Reply:
You got it